src/gateways/internal-api.test.ts

```diff
@@ -14,7 +14,7 @@ const mockedAxios = axios as jest.Mocked<typeof axios>;
 describe('Internal API Gateway', () => {
   const gateway = new InternalApiGateway();
 
-  describe('getEvidenceRequests', () => {
+  xdescribe('getEvidenceRequests', () => {
```

Comment:
Should this describe block be deleted since it's not being used?

Reply:
We'll need it once the API endpoint is done and we can revert the stubbing in the method